Makefile

```diff
@@ -34,4 +34,6 @@ ci.lint-erb: ## Run the ERB linter
 
 .PHONY: ci.test
 ci.test: ## Run the tests with results formatted for CI
-	docker-compose run --rm web /bin/sh -c 'bundle exec --verbose rspec --format RspecJunitFormatter' > rspec-results.xml
+	docker-compose run --rm web /bin/sh -c 'apk add nodejs yarn && \
+		bundle exec rails assets:precompile && \
```

Comment:
Interesting. it shouldn't be necessary to precompile in order to run the specs.

Reply:
I just double checked on this one and this is the error I get when I remove the assets:precompile option.
```
--- Caused by: ---
Webpacker::Manifest::MissingEntryError:
  Webpacker can&apos;t find media/images/favicon.ico in /app/public/packs-test/mm
anifest.json. Possible causes:
  1. You want to set webpacker.yml value of compile to true for your environment
     unless you are using the `webpack -w` or the webpack-dev-server.
  2. webpack has not yet re-run to reflect updates.
  3. You have misconfigured Webpacker&apos;s config/webpacker.yml file.
  4. Your webpack configuration is not creating a manifest.
```
I assumed (2) was the cause, hence the presence of the precompile option. Should I have done something else to resolve this?

Reply:
It's clearly using the test env as the path points to `packs-test`. As I understand the out of the box behaviour in test env it dynamically builds the webpacker assets when they're first released. I have no idea why this isn't working in quite that way, though. Don't see much point in blocking on this